Userland/Libraries/LibWasm/AbstractMachine/AbstractMachine.cpp

```diff
@@ -399,12 +406,6 @@ Optional<InstantiationError> AbstractMachine::allocate_all_initial_phase(Module
 
     // FIXME: What if this fails?
```

Comment:
More of a question, should this FIXME be moved with the above code to "AbstractMachine::instantiate" as well, or does it apply to something else?

I feel like it should be moved (it applies to the VERIFY inside of the moved code), but not sure.

Reply:
:thinking: I personally interpreted that as "what if something in this function fails", I'll take a look again and move that comment if was referring to the verify.